server/lib/url.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const fetchHead = require('./wrap-fetch')(require('@quarterto/fetch-head'), {
+const fetch = require('./wrap-fetch')(require('node-fetch'), {
```

Comment:
I'm using the body now: https://github.com/Financial-Times/google-amp/pull/156/files#diff-55118bc42fe8265825e9d4821e462d82R46